tracim/command/database.py

```diff
@@ -46,6 +46,7 @@ def take_action(self, parsed_args: argparse.Namespace) -> None:
         config_uri = parsed_args.config_file
         setup_logging(config_uri)
         settings = get_appsettings(config_uri)
+        settings.update(settings.global_conf)
```

Comment:
this requires at least a comment in order to explain to developers why settigns from global_conf are merged into settings.